test/test_nestedtensor.py

```diff
@@ -3660,6 +3660,38 @@ def check_nt_equality(x, y):
         self.assertFalse(clone.is_contiguous())
         check_nt_equality(detached, transposed)
 
+    def test_to_copy(self, device):
+        nt, _ = jagged_from_list(
+            [torch.randn(i + 2, 3, 4, requires_grad=True, dtype=torch.float64, device=device) for i in range(3)], None
+        )
+
+        nt_copy_dtype = torch.ops.aten._to_copy(nt, dtype=torch.float16)
+        self.assertEqual(torch.float16, nt_copy_dtype.dtype)
+
+        nt_t = nt.transpose(1, 2)
```

Comment:
This isn't a comprehensive test (since it's a particular kind of ragged_idx != 1 instance), but I think it's fine for the purposes of this work.